docs/root/faq/zone_aware_routing.rst

```diff
@@ -12,42 +12,48 @@ This section describes the specific configuration for the Envoy running side by
 These are the requirements:
 
 * Envoy must be launched with :option:`--service-zone` option which defines the zone for the current host.
-* Both definitions of the source and the destination clusters must have :ref:`sds <config_cluster_manager_type>` type.
-* :ref:`local_cluster_name <config_cluster_manager_local_cluster_name>` must be set to the source cluster.
+* Both definitions of the source and the destination clusters must have :ref:`EDS <envoy_api_field_Cluster.type>` type.
+* :ref:`local_cluster_name <envoy_api_field_config.bootstrap.v2.ClusterManager.local_cluster_name>` must be set to the
+  source cluster.
 
   Only essential parts are listed in the configuration below for the cluster manager.
 
 .. code-block:: json
 
   {
-    "sds": "{...}",
-    "local_cluster_name": "cluster_a",
-    "clusters": [
-      {
-        "name": "cluster_a",
-        "type": "sds",
-      },
-      {
-        "name": "cluster_b",
-        "type": "sds"
-      }
-    ]
+    "cluster_manager": {
+      "local_cluster_name": "cluster_a"
```

Comment:
Can you split out these non-mechanical changes to a separate PR? It's easier to review vs. the bulk sed.

Reply:
Right, sorry for that. I'll extract them to a new PR.

Reply:
I think I finished remove all of non-mechanical changes from this PR.